src/Interfaces/IOpdexVault.cs

```diff
@@ -51,33 +82,73 @@ public interface IOpdexVault
     void NotifyDistribution(UInt256 amount);
 
     /// <summary>
-    /// Allows the vault owner to issue new certificates to wallet addresses.
+    /// Redeems a vested certificate the sender owns and transfers the claimed tokens.
+    /// </summary>
+    void RedeemCertificate();
+
+    /// <summary>
+    /// Creates a new proposal proposing a certificate be created.
+    /// </summary>
+    /// <param name="amount">The amount of tokens proposed the recipient's certificate be assigned.</param>
+    /// <param name="recipient">The recipient address to assign a certificate to if approved.</param>
+    /// <param name="description">A description of the proposal, limited to 200 characters, preferably a link.</param>
+    /// <returns>The Id of the generated proposal.</returns>
+    ulong CreateNewCertificateProposal(UInt256 amount, Address recipient, string description);
+
+    /// <summary>
+    /// Creates a new proposal proposing a certificate be revoked.
+    /// </summary>
+    /// <param name="recipient">The recipient of an existing certificate to have revoked.</param>
+    /// <param name="description">A description of the proposal, limited to 200 characters, preferably a link.</param>
+    /// <returns>The Id of the generated proposal.</returns>
+    ulong CreateRevokeCertificateProposal(Address recipient, string description);
+
+    /// <summary>
+    /// Creates a new proposal proposing a new pledge minimum amount of tokens have pledged to move onto a vote.
+    /// </summary>
+    /// <param name="amount">The proposed new minimum pledge amount.</param>
+    /// <param name="description">A description of the proposal, limited to 200 characters, preferably a link.</param>
+    /// <returns>The Id of the generated proposal.</returns>
+    ulong CreatePledgeMinimumProposal(UInt256 amount, string description);
+
+    /// <summary>
+    /// Creates a new proposal proposing a new proposal minimum amount of tokens have voted to be considered for approval.
+    /// </summary>
+    /// <param name="amount">The proposed new minimum proposal amount.</param>
+    /// <param name="description">A description of the proposal, limited to 200 characters, preferably a link.</param>
+    /// <returns>The Id of the generated proposal.</returns>
+    ulong CreateProposalMinimumProposal(UInt256 amount, string description);
+
+    /// <summary>
+    /// Pledge for a proposal by temporarily locking CRS tokens to help meet the minimum pledge amount to move to an official vote.
```

Comment:
```suggestion
    /// Pledge for a proposal by temporarily locking CRS tokens to help meet the minimum pledge amount, so that it can move to an official vote.
```